docs/pages/menubar.mdx

```diff
@@ -169,3 +169,47 @@ Default trigger - `hover`
   <MenuBarItem>Blog</MenuBarItem>
 </MenuBar>
 ```
+
+### mode and isCollapsable
+
+Default mode - `horizontal`
+
+Default isCollapsable - `false`
+
+```jsx
+<MenuBar mode="vertical" trigger="click">
```

Comment:
```suggestion
<MenuBar mode="vertical" trigger="click" maxW={300}>
```

We can include the maxWidth to demonstrate the vertical menubar for example here